ethereum/chain.py

```diff
@@ -1,4 +1,5 @@
 import time
+from itertools import count, islice
```

Comment:
I love nitpicks! This is a great point--changing it now. I guess the only tradeoff is you don't know explicitly what is being used from `itertools`. However, with a function name as generic as `count()` I totally agree it's a bad idea to not use `itertools.count`